core.js

```diff
@@ -0,0 +1,685 @@
+/**
+ * Spacebrew Manager
+ * @module spacebrew/manager
+ */
+
+var Leaf = require('./leaf.js');
+var Route = require('./route.js');
+var Admin = require('./admin.js');
+
+/**
+ * Creates an instance of a Spacebrew Manager
+ * @constructor
+ */
+var Manager = function(){
+  /** the clients registered with this network */
+  this.clients = [];
+  /** the routes configured on this network */
+  this.routes = [];
+  /** the admins registered with this network */
+  this.admins = [];
+  /** the other managers in this network */
+  this.managers = [];
+};
+
+/**
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Missing JSDoc parameter type for 'client'.](https://www.codacy.com/app/quinkennedy/spacebrew/file/2279533613/issues/source?bid=3184377&fileBranchId=3187330#l25)